content/_index.de.md

```diff
@@ -0,0 +1,62 @@
+
+Für uns als Industrie ist es an der Zeit, die Idee hinter uns zu lassen, dass das Verdienst - und deren Anerkennung - etwas sei, dass gemessen, von jedem Individuum  unter denselben Bedingungen angestrebt und jemals fair verteilt werden kann.
+
+Wie sieht eine postmeritokratische Welt aus? Sie beruht auf grundlegenden Werten und Prinzipien, einer Bejahung des Dazugehörens, die auf alle Menschen zutrifft, die die Entwicklung von Software praktizieren.
+
+### Unsere Werte
+
+Diese grundlegenden Werte und Prinzipien sind:
+
+* Wir glauben nicht daran, dass unser Wert als Menschen intrinsisch an unseren Wert als Wissensarbeiter*innen gebunden ist. Unsere Berufe definieren uns nicht; wir sind mehr als die Arbeit, die wir ausüben.
+
+* Wir glauben daran, dass zwischenmenschliche Fähigkeiten mindestens genau so wichtig sind wie technische Fähigkeiten.
+
+* Wir können als professionelle Entwickler*innen den größten Beitrag leisten, indem wir aus der Diversität unserer Identitäten, Hintergründe, Erfahrungen und Perspektiven schöpfen. Homogenität ist ein Anti-Pattern.
+
+* Wir können erfolgreich sein und gleichzeitig vielfältige, erfüllende Leben führen. Unser Erfolg und unser Wert ist nicht davon abhängig, ob wir all unsere Energie in die Entwicklung von Software stecken.
+
+* We can be successful while leading rich, full lives. Our success and value is not dependent on exerting all of our energy on contributing to software.
+
+* Wir haben die Verpflichtung unsere - wie auch immer subtil - privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
+
+* Wir müssen Raum schaffen für Menschen, die nicht so sind wir, damit sie in unseren Bereich einsteigen und darin erfolgreich sind. Das bedeutet nicht nur, diese Menschen in den IT-Bereich einzuladen, sondern auch sicherzustellen, dass sie darin Unterstützung und Ermächtigung finden.
+
+* Wir haben eine moralische Verantwortung die Arbeit an Software zu verweigern, die das Wohl anderer Menschen negativ beeinträchtigt.
+
+* Wir erkennen den Wert von Mitwirkenden, die nicht-technische Beiträge leisten, als gleichwertig zu dem Wert Mitwirkender an, die technische Beiträge liefern.
+
+* Wir verstehen, dass die Arbeit in unserem Feld kein Recht, sondern ein Privileg ist. Der negative Einfluss toxischer Menschen am Arbeitsplatz und in der darüber hinausgehenden Gemeinschaft wird nicht durch deren technischen Beiträge wieder wettgemacht.
+
+* Wir widmen uns der Ausübung von Mitgefühl, anstelle von Verachtung. Wir weigern uns jedoch, andere Menschen aufgrund der Wahl ihrer Werkzeuge, Methoden oder Programmiersprachen herunterzumachen.
+
+* Der IT-Bereich begrüßt technischen Wandel und wird bereichert, indem es auch sozialen Wandel akzeptiert.
+
+* Wir streben danach, unsere Werte in allem, was wir tun, widerspiegeln zu lassen. Wir sind uns darüber im Klaren, dass Werte, die angepriesen, aber nicht in die Tat umgesetzt werden, in erster Linie gar keine Werte sind.
+
+
+### Unterzeichner*innen
```

Comment:
In general a lot of personal nouns will have a male (often ending in `-` or `er`, e.g. `Unterzeichner`) and a dedicated female version (often ending in `-` or `erin`, e.g. `Unterzeichnerin`) in German which is often used to describe a group of people of a particular gender. To describe groups of which the gender is not known oftentimes the _Binnen-I_ (e.g. `UnterzeichnerInnen`) is used to address people who either identify as male and female. Although this practice is very common, it's not inclusive to non-binary people imho. 

Therefore I decided to use the asterisk `*` here which can be used to infer, that not only people of male or female gender, but also people who identify with other genders are addressed. You can also read more about it in this [Wiki Article about gender-neutral language in German](https://nonbinary.miraheze.org/wiki/Gender_neutral_language_in_German):

>Nouns, Adjectives etc. [...]
Asterisk, Underscore, Punctuation
Pronounced as a pause or glottal stop. In between "feminine" and "masculine" ending, the most common version:

> mein_e beste*r Freund:in
Either after the stem - or more realistically at the next best place that has nothing to do with feminine/masculine forms but makes sense for pronunciation:

> mei:ne bes_ter Freun*din
At random places:

> mei*ne best:er Freundi_n
Can be used consistently or mixed up like in the example above.

Reply:
That seems like a great approach, thank you for your sensitivity here.

Reply:
Yes, sure!